2_flow_metrics.R

```diff
@@ -101,6 +103,64 @@ p2_targets_list<- list(
                   grep(colnames(p2_FDC_metrics_low), pattern = 'q0.01'))
              ]
   ),
+  ##Compute FDC-based metrics using non-overlapping event definitions
+  tar_target(p2_FDC_metrics_nonoverlapping,
+             calc_FDCmetrics(site_num = p1_screened_site_list, 
+                             clean_daily_flow = p1_clean_daily_flow, 
+                             yearType = yearType,
+                             drainArea_tab = p1_drainage_area,
+                             NE_probs = NE_quants,
+                             seasonal = FALSE,
+                             year_start = year_start,
+                             out_format = 'pivot',
+                             allow_event_overlap = FALSE),
+             map(p1_screened_site_list),
+             deployment = 'worker'
+  ),
+  
+  tar_target(p2_FDC_metrics_low_nonoverlapping,
+             calc_FDCmetrics(site_num = p1_screened_site_list, 
+                             clean_daily_flow = p1_clean_daily_flow, 
+                             yearType = yearType,
+                             drainArea_tab = p1_drainage_area,
+                             NE_probs = NE_quants_low,
+                             seasonal = FALSE,
+                             year_start = year_start,
+                             out_format = 'pivot',
+                             threshold_type = 'low',
+                             allow_event_overlap = FALSE),
+             map(p1_screened_site_list),
+             deployment = 'worker'
+  ),
+  #For these aggregated ranges of events, we would not be interested in the
+  # category above the second NE_quants (70-max in this case). Should remove
+  # that result from these targets.
```

Comment:
@slevin75: I added these 2 example targets that span a larger range than specified in NE_quants. If it looks like we should still aggregate quantiles in the cluster analysis, I'd like to compare cluster results for this method vs. using each individual quantile (as in the above targets). But there may be a better way to do this because we wouldn't need the results for the second NE_quant value. See my comment in the script above this target.